apps/site/lib/site_web/controllers/route_controller.ex

```diff
@@ -6,7 +6,30 @@ defmodule SiteWeb.RouteController do
   alias Routes.Repo
 
   def get_by_stop_id(conn, %{"stop_id" => stop_id} = _params) do
-    routes = Repo.by_stop(stop_id)
-    json(conn, routes)
+    routesWithPolylines =
+      stop_id
+      |> Repo.by_stop()
+      |> Enum.map(&[&1, route_polylines(&1)])
```

Comment:
suggestion: you probably want to include the polylines underneath the route object, rather than switching this from a list of objects to a list of arrays. In this situation, a older frontend client that hits a newer backend server will crash, rather than silently ignoring the new data.
```suggestion
      |> Enum.map(&Map.put(&1, :polylines, route_polylines(&1))
```

Reply:
The suggested line didn't work as-is but I was able to make the change, I like it much better, thanks!